common/constants/rmq-constants.ts

```diff
@@ -0,0 +1 @@
+export const FILE_SERVICE = "preSignedUrl"
```

Comment:
Can be replaced with `"FILE_UPLOADER_SERVICE"`.
Also, can be moved to `src/files/constants.ts` file